tests/e2e/tests/regression/browser/stream-key.e2e.ts

```diff
@@ -0,0 +1,110 @@
+import { Chance } from 'chance';
+import { acceptLicenseTermsAndAddDatabase, deleteDatabase } from '../../../helpers/database';
+import { rte } from '../../../helpers/constants';
+import { BrowserPage, CliPage } from '../../../pageObjects';
+import { commonUrl, ossStandaloneConfig } from '../../../helpers/conf';
+
+const browserPage = new BrowserPage();
+const cliPage = new CliPage();
+const chance = new Chance();
+
+const field = chance.word({length: 5});
+const value = chance.word({length: 5});
+let keyName = chance.word({length: 20});
```

Comment:
I see you like to use it, you can create a function in helped which generates it and send only the length. Would be essentials for you